package.json

```diff
@@ -16,7 +16,8 @@
     "rollup": "^2.77.2",
     "rollup-plugin-dts": "^4.2.2",
     "rollup-plugin-esbuild": "^4.10.2",
-    "turbo": "^1.8.3"
+    "turbo": "^1.8.3",
+    "lodash": "https://github.com/lodash/lodash.git#npm"
```

Comment:
What are you using lodash for? I don't see any usage with it. 

and if it's only used in `apps/phone`, it should be installed in that package.json

Reply:
It's a dependency for eslint and a few other packages in the dev dependency. The dependency downloaded by npn is broken and doesn't have the modules required so the build will just fail